Documentation/gettingstarted/k8s-install-openshift-okd.rst

```diff
@@ -118,54 +118,40 @@ Next, generate OpenShift manifests:
 
    openshift-install create manifests --dir "${CLUSTER_NAME}"
 
-Now, define ``cilium`` namespace:
+Next, obtain Cilium manifest from ``cilium/cilium-olm`` repository and copy to ``${CLUSTER_NAME}/manifests``:
 
-.. code:: bash
+.. parsed-literal::
 
-   cat << EOF > "${CLUSTER_NAME}/manifests/cluster-network-03-cilium-namespace.yaml"
-   apiVersion: v1
-   kind: Namespace
-   metadata:
-     name: cilium
-     annotations:
-       # node selector is required to make cilium-operator run on control plane nodes
-       openshift.io/node-selector: ""
-     labels:
-       name: cilium
-       # run level sets priority for Cilium to be deployed prior to other components
-       openshift.io/run-level: "0"
-       # enable cluster logging for Cilium namespace
-       openshift.io/cluster-logging: "true"
-       # enable cluster monitoring for Cilium namespace
-       openshift.io/cluster-monitoring: "true"
-   EOF
+   cilium_olm_rev=master
 
-.. include:: k8s-install-download-release.rst
+   curl --silent --location --fail --show-error "https://github.com/cilium/cilium-olm/archive/${cilium_olm_rev}.tar.gz" --output /tmp/cilium-olm.tgz
+   tar -C /tmp -xf /tmp/cilium-olm.tgz
```

Comment:
For the main Cilium instructions, we have had instructions like this for developers (ie `latest` docs only, not for the branch docs). Is that the case here? Is it normal to use instructions like this for a plugin install into a OpenShift environment?

Reply:
Yes, this is mostly normal for installing software to OpenShift during bootstrap. At least one other networking "plugin" does a very similar thing.
User needs to pass manifests for the operator that installs Cilium, the most convent way to handle this was to check rendered manifest in the cilium-olm repo. What is being installed is an operator (for now we are using RedHat's Helm operator, but going forward this will become our own installer operator).
The reason master branch is referenced here is just to avoid having to update the docs each time there is a minor , user might choose to pin to a commit. The cilium-olm repo doesn't not have release branches, but it does contain manifests for most of 1.8.x releases, all of 1.9.x release and all others going forward. There is more nuance to this, but that's the idea overall.